apps/etl-func/src/domain/message-status.ts

```diff
@@ -0,0 +1,37 @@
+import * as z from "zod";
+
+const rejectionReasonSchema = z
+  .enum(["SERVICE_NOT_ALLOWED", "USER_NOT_FOUND", "UNKNOWN"])
+  .default("UNKNOWN");
+
+const rejectedMessageStatusSchema = z.object({
+  fiscalCode: z.string().min(1).optional(),
```

Comment:
```suggestion
  fiscalCode: z.string().min(1),
```
Why don't we set the fiscalCode as mandatory? In the old schema it was optional because of backwards compatibility for older MessageStatus. We decided that the ingestion for the messageStatus will not have older items so I think it is better to have it mandatory.

Reply:
You are right we could make it mandatory in this case

Reply:
can we use fiscalCode: fiscalCodeSchema instead ?

Reply:
[0530efe](https://github.com/pagopa/io-messages/pull/59/commits/0530efe8115fb8a44f61fe63cb658b2be506ad3d)